scripts/paperless-webserver.service

```diff
@@ -9,7 +9,21 @@ Requires=redis.service
 User=paperless
 Group=paperless
 WorkingDirectory=/opt/paperless/src
-ExecStart=python3 webserver.py
+
+Environment=GRANIAN_HOST=::
+Environment=GRANIAN_PORT=8000
+Environment=GRANIAN_WORKERS=1
+
+ExecStart=/bin/sh -c '\
+  # Host: GRANIAN_HOST -> PAPERLESS_BIND_ADDR -> default \
+  [ -z "$GRANIAN_HOST" ] && [ -n "$PAPERLESS_BIND_ADDR" ] && export GRANIAN_HOST=$PAPERLESS_BIND_ADDR; \
+  # Port: GRANIAN_PORT -> PAPERLESS_PORT -> default \
+  [ -z "$GRANIAN_PORT" ] && [ -n "$PAPERLESS_PORT" ] && export GRANIAN_PORT=$PAPERLESS_PORT; \
+  # Workers: GRANIAN_WORKERS -> PAPERLESS_WEBSERVER_WORKERS -> default \
+  [ -z "$GRANIAN_WORKERS" ] && [ -n "$PAPERLESS_WEBSERVER_WORKERS" ] && export GRANIAN_WORKERS=$PAPERLESS_WEBSERVER_WORKERS; \
```

Comment:
How does this still fallback to the default (1), maybe Im missing how that happens

Reply:
Yeah, I think that was wrong.  What I want is if `PAPERLESS_` is defined (non-zero length), change `GRANIAN_` version to that value and export it.  So if not defined, they'll take the defaults set above